lib/rucio/daemons/bb8/common.py

```diff
@@ -112,28 +132,46 @@ def __dump_url(rse_id, logger=logging.log):
     # get the date of the most recent dump
     today = date.today()
     dump_dates = []
-    dump_production_day = config_get('bb8', 'dump_production_day', raise_exception=False, default=None)
+    dump_production_day = config_get(
+        "bb8", "dump_production_day", raise_exception=False, default=None
+    )
     if dump_production_day is None:
         for idx in range(0, 7):
             dump_date = today - timedelta(idx)
-            dump_dates.append(dump_date.strftime('%d-%m-%Y'))
+            dump_dates.append(dump_date.strftime("%d-%m-%Y"))
     else:
-        weekdays = {'Sunday': 6, 'Monday': 0, 'Tuesday': 1, 'Wednesday': 2, 'Thursday': 3, 'Friday': 4, 'Saturday': 5}
+        weekdays = {
+            "Sunday": 6,
+            "Monday": 0,
+            "Tuesday": 1,
+            "Wednesday": 2,
+            "Thursday": 3,
+            "Friday": 4,
+            "Saturday": 5,
+        }
         if dump_production_day not in weekdays:
-            logger(logging.WARNING, 'ERROR: please set the day of a dump creation in bb8 config correctly, e.g. Monday')
+            logger(
+                logging.WARNING,
+                "ERROR: please set the day of a dump creation in bb8 config correctly, e.g. Monday",
+            )
             return False
         today_idx = (today.weekday() - weekdays[dump_production_day]) % 7
         dump_date = today - timedelta(today_idx)
-        dump_dates = [dump_date.strftime('%d-%m-%Y')]
+        dump_dates = [dump_date.strftime("%d-%m-%Y")]
 
     # getting structure (template) of url location of a dump
-    url_template_str = config_get('bb8', 'dump_url_template', raise_exception=False, default='http://rucio-analytix.cern.ch:8080/LOCKS/GetFileFromHDFS?date=${date}&rse=${rse}')
+    url_template_str = config_get(
+        "bb8",
+        "dump_url_template",
+        raise_exception=False,
+        default="http://rucio-analytix.cern.ch:8080/LOCKS/GetFileFromHDFS?date=${date}&rse=${rse}",
```

Comment:
is this supposed to be the real URL? because then it's wrong. :-)

Reply:
Remove default